kolibri/core/assets/src/views/AppError/index.vue

```diff
@@ -0,0 +1,109 @@
+<template>
```

Comment:
Let's delete the ErrorBox component.

Reply:
Ah. A bunch of files I'd deleted came back in the merge. Good catch.